server/core/src/https/views/login.rs

```diff
@@ -61,35 +63,50 @@ pub async fn view_index_get(
     VerifiedClientInformation(client_auth_info): VerifiedClientInformation,
     Extension(kopid): Extension<KOpId>,
     _jar: CookieJar,
-) -> Response {
+) -> axum::response::Result<Response> {
     // If we are authenticated, redirect to the landing.
-    let session_valid_result = state
+    let auth_result = state
         .qe_r_ref
         .handle_auth_valid(client_auth_info, kopid.eventid)
         .await;
 
-    match session_valid_result {
-        Ok(()) => {
-            // Send the user to the landing.
-            Redirect::to("/ui/apps").into_response()
-        }
-        Err(OperationError::NotAuthenticated) | Err(OperationError::SessionExpired) => {
-            // cookie jar with remember me.
-
-            HtmlTemplate(LoginView {
-                username: "",
-                remember_me: false,
-            })
-            .into_response()
+    // manual error handling because we don't want to see a not logged/session expired error
+    // on the login page
+    match auth_result {
+        Ok(_) => Ok(Redirect::to("/ui/apps").into_response()),
+        Err(err) => {
+            match err {
+                OperationError::NotAuthenticated | OperationError::SessionExpired => {
```

Comment:
These are impossible states at this point, so instead of making an infinite login loop, render the error as it was before. (Better yet, undo this whole section of your pr).

Reply:
(I say that mostly because I think the whole thing is going to conflict with the PR I just put in, and I think there are better ways to fix the logout issue potentially, so I'll do that in a follow up PR. )